### ISSUE TITLE
wrong stacking of firewood in clay oven

### ISSUE THREAD

**Issue by JustForVS** [anegostudios/VintageStory-Issues#2207]
**Game Version:** 1.17.4  
**Platform:** Windows 10   
**Modded:** Yes (hudclock-2.8.0, ProspectorInfo-3.2.0, singlepause-1.0.3, ZoomButton1.3.0)  
**SP/MP:** Singleplayer

### Description
After Patch to 1.17.4 firewood is stacked wrong in clay oven, when opening is directing to west or east (see screen shot).
Happens in creative mode, too.

### How to reproduce
1. Make a clay oven
2. Set clay oven with opening directing to west or east (according to world map) and fill it with firewood
3. firewood is stacked wrong

### Expected behavior
If not obvious, what did you expect to happen?

### Screenshots
Attach a screenshot if it helps explain the issue / shows a visual bug.

### Logs
```
Paste crash between the triple backticks.

In the game's Logs folder you may be able to find more information
inside client-main.txt and server-main.txt. Alternatively, you may use
gist.github.com to upload logs, linking them here, if they're too large.
```

**Comment by JustForVS** [comment]
![Unbenannt](https://user-images.githubusercontent.com/87942616/191543501-a68e6d2e-ac81-4eba-862d-e72b6ac68726.jpg)

**Comment by Saricane** [comment]
I can confirm.

**Comment by Zamaky345** [comment]
I can confirm this too and ingots are stacking wrong on shelfs too, might be the same problem.

**Comment by Craluminum2413** [comment]
@Zamaky345 There is an issue for ingots on shelves #1773

**Comment by Zamaky345** [comment]
> @Zamaky345 There is an issue about ingots on shelves #1773

Thanks for letting me know o/

**Comment by Zamaky345** [comment]
@Craluminum2413 
thanks for letting me know

**Comment by JustForVS** [comment]
Still happens when loading a game and oven was already filled with firewood (not burning).
As soon as firewood is burning, it is stacked correct.

**Comment by kurazarrh** [comment]
Can confirm @JustForVS 's comment; stacking logs in the oven appears correctly now, but after closing & reloading a save, the logs face the wrong way, and lighting them on fire corrects the orientation.

**Comment by KxTxLiNn** [comment]
Can confirm it's still happening in 1.17.9.

**Comment by Craluminum2413** [comment]
Fixed since 1.18.0-pre.1
